### PR TITLE
Added Inline Documentation to ID Initializer

### PR DESCRIPTION
**Pull request by calebkleveter**
Closes https://github.com/vapor/fluent-kit/issues/359

### CODE REVIEW

File: Sources/FluentKit/Properties/ID.swift [hunk 1]
```diff
@@ -56,6 +56,12 @@ public final class IDProperty<Model, Value>
         }
     }
 
+    /// Initializes an `ID` property with the key `.id` and type `UUID`.
+    ///
+    /// If the property's type is not `UUID` or the key is not `.id`, the initializer will
+    /// fatal error. This is so Fluent can support databases like MonogoDB more natively.
+    ///
+    /// If you want to specify a custom ID key or type, use the `.init(custom:generatedBy:)` initializer.
```
Comment (0xTim) [inline Sources/FluentKit/Properties/ID.swift:64]: Nit - tighten up the language
```suggestion
    /// If you want to specify a custom ID key or type, use `.init(custom:generatedBy:)`.
```

File: Sources/FluentKit/Properties/ID.swift [hunk 2]
```diff
@@ -56,6 +56,12 @@ public final class IDProperty<Model, Value>
         }
     }
 
+    /// Initializes an `ID` property with the key `.id` and type `UUID`.
+    ///
+    /// If the property's type is not `UUID` or the key is not `.id`, the initializer will
+    /// fatal error. This is so Fluent can support databases like MonogoDB more natively.
```
Comment (0xTim) [inline Sources/FluentKit/Properties/ID.swift:62]: Nit - tighten up the language
```suggestion
    /// fatal error. This is so Fluent can natively support databases like MonogoDB.
```

**Review by 0xTim — CHANGES_REQUESTED** [review]
Couple of tiny language nits

File: Sources/FluentKit/Properties/ID.swift (hunk 1 above)
Comment (gwynne) [reply Sources/FluentKit/Properties/ID.swift:64]: Double nit:
```suggestion
    /// Use the `.init(custom:generatedBy:)` initializer to specify a custom ID key or type.
```

File: Sources/FluentKit/Properties/ID.swift (hunk 2 above)
Comment (gwynne) [reply Sources/FluentKit/Properties/ID.swift:62]: Double nit (and spelling!):
```suggestion
    /// fatal error. This allows Fluent to natively support databases like MongoDB.
```

**Review by gwynne — APPROVED** [review]

**Review by gwynne — CHANGES_REQUESTED** [review]
Wrong button, I meant Request changes 😞

**Review by 0xTim — APPROVED** [review]
LGTM

**Review by gwynne — APPROVED** [review]

**Pull request merged**